### DISCUSSION TITLE
Use floating IP addresses for nodes

### DISCUSSION THREAD

**Discussion by dclayton77** [vitobotta/hetzner-k3s#329] [Q&A]
During cluster creation, is it possible to configure Hetzner to use Floating IP addresses for nodes?  The reason is I use whitelisting of the 
 node IP address for access to Azure services, Kong Whitelisting etc.   For now, I have the IP hardcoded but when it changes, I need to update and redeploy several services.  If I could use a Floating IP, I believe that I could use that for whitelisting a node and then it would not matter if the actual IP address changes.

Ideally this would be done as part of the YAML configuration as I am trying to avoid any manual intervention.

**Comment by vitobotta** [comment]
I believe I replied to the issue already but for others: this requires a big effort for a very specific use case, so I am not planning on working on this for the time being.

**Reply by dclayton77** [reply]
No problem, as you mentioned in the related issue, I will look at using a proxy instead.
